ohara-manager/client/src/components/Pipeline/PipelineHooks.js

```diff
@@ -69,7 +69,7 @@ export const usePipelineState = () => {
       case 'updateMetrics': {
         return {
           ...state,
-          objects: [...state.objects, ...payload],
+          objects: [...payload],
```

Comment:
We could just change this to:

```suggestion
          objects: payload,
```

Reply:
thanks nice point